Feat/4039  make public profile personal page editable (#4086)
tracim/tracim: backend/doc/user_custom_properties.md

### Tracim specific properties in ui schemas

#### The display_group property

> This property is mandatory.

Reviewer: ```suggestion
This property is mandatory for properties to appear in the Tracim user interface.
```

The frontend have two forms to display in the public profile page: Information and Personal data.  
The data from the both forms come from the single json schema in backend.  
The frontend then have to split the schema for each form.  
To do so, the ui schema has a property to specify on which form, a given property belong to:  

    tracim:display_group

The frontend currently expects two possible values:

> - `public_profile_first`
> - `public_profile_second`

Reviewer 2: ```suggestion
- `public_profile_first`: to display at Information
- `public_profile_second`: to display at Personal data
```

Reviewer 3: at X → in the X box

> The frontend have two forms to display in the public profile page: Information and Personal data.  
> The data from the both forms come from the single json schema in backend.  
> The frontend then have to split the schema for each form.  
> To do so, the ui schema has a property to specify on which form, a given property belong to:  
> 
>     tracim:display_group
> 
> The frontend currently expects two possible values:
> - `public_profile_first`
> - `public_profile_second`

Reviewer: ```suggestion
The Tracim user interface has two forms where user properties can be displayed in the public profile page: Information and Personal data.
Each user property can be assigned to one or the other form using the special key `tracim:display_group` in the UI schema file.
To display a property in the Information column, use `"tracim:display_group": "public_profile_first"`. To display a property in the `Personal data` column, use `"tracim:display_group": "public_profile_second"`.
```

> see [ui schema](../examples/user_custom_properties/organization/ui.json) example

Reviewer 3: See [ui schema](../examples/user_custom_properties/organization/ui.json) for an example.
